Sources/TensorFlow/Loss.swift

```diff
@@ -228,6 +246,8 @@ func _vjpSoftmaxCrossEntropyHelper<Scalar: TensorFlowFloatingPoint>(
 }
 
 /// Returns the softmax cross entropy (categorical cross entropy) between logits and labels.
+/// Given the logits and probabilites, the softmax cross entropy computes `reduction(-softmax(logits) * log(p))`
+/// Where softmax(x) = `exp(x)/sum(exp(x))`
```

Comment:
For consistency since you added spaces b/w the operators in changes above:
```suggestion
/// Where softmax(x) = `exp(x) / sum(exp(x))`
```